From 6d9922586d708fcfcd17606bbef535165bbd2a0d Mon Sep 17 00:00:00 2001
From: Daniel Jarai <jaraidaniel@gmail.com>
Date: Wed, 25 Jul 2018 14:54:40 +0200
Subject: [PATCH] Fix the HIDE_NON_SYSTEM_OVERLAY_WINDOWS permission definition

Change Ic51ead670fc480e549512ba1d02f49d9c13bc3f0 incorrectly tries making
this permission hidden.
While marking it with @hide and only adding it to api/system-current.txt
was the proper way on Android 6.0, on 4.4, we need to add system APIs to
api/current.txt, without hiding them.

This change adapts this permission definition for 4.4.

Change-Id: Ib2de017d396aa9dcefa91f7eb1ea210be668234d
---
 core/res/AndroidManifest.xml | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/core/res/AndroidManifest.xml b/core/res/AndroidManifest.xml
index eb87fb5..e3bdf46 100644
--- a/core/res/AndroidManifest.xml
+++ b/core/res/AndroidManifest.xml
@@ -1843,12 +1843,10 @@
         android:description="@string/permdesc_internalSystemWindow"
         android:protectionLevel="signature" />
 
-    <!-- @SystemApi Allows an application to use
+    <!-- Allows an application to use
         {@link android.view.WindowManager.LayoutsParams#PRIVATE_FLAG_HIDE_NON_SYSTEM_OVERLAY_WINDOWS}
         to hide non-system-overlay windows.
-        <p>Not for use by third-party applications.
-        @hide
-    -->
+        <p>Not for use by third-party applications. -->
     <permission android:name="android.permission.HIDE_NON_SYSTEM_OVERLAY_WINDOWS"
         android:protectionLevel="signature|system" />
 
-- 
2.7.4

